.circleci/config.yml

```diff
@@ -203,34 +148,51 @@ workflows:
   version: 2
   test:
     jobs:
-      - test
+      - test-unit
   publish-linux:
     jobs:
-      - test:
+      - test-unit:
           filters:
             tags:
               only: /[0-9]+\.[0-9]+\.[0-9]+/
             branches:
               ignore: /.*/
-      - linux-build:
+      - build-linux:
           requires:
-            - test
+            - test-unit
+          filters:
+            tags:
+              only: /[0-9]+\.[0-9]+\.[0-9]+/
+            branches:
+              ignore: /.*/
+      - test-linux:
+          requires:
+            - build-linux
+          filters:
```

Comment:
Do we need to repeat filters if we're already mentioning that `test-linux` build requires `build-linux`?

Reply:
we may not but for keeping things safe and there is no harm in doing this, I'd say let's keep them

Reply:
hrmmm I don't see any harm in removing it but yeah for now I'll approve changes.